Weird compiler warning in segmentation module

I'm compiling with normal flags, GCC 9.1 (Linux) and am looking at the following warning:
```c++
segmentation/include/pcl/segmentation/supervoxel_clustering.h: In member function ‘void pcl::octree::OctreePointCloudAdjacencyContainer<PointInT, DataT>::reset() [with PointInT = pcl::PointXYZRGBA; DataT = pcl::SupervoxelClustering<pcl::PointXYZRGBA>::VoxelData]’:
segmentation/include/pcl/segmentation/supervoxel_clustering.h:134:13: warning: ‘*((void*)&<anonymous> +52)’ is used uninitialized in this function [-Wuninitialized]
  134 |       class VoxelData
      |             ^~~~~~~~~
```

This is a class inside another class, which ideally should not trip up compilers.

With GCC 7.4 I get a different, more helpful, warning:

```
../segmentation/include/pcl/segmentation/supervoxel_clustering.h: In member function ‘pcl::octree::OctreePointCloudAdjacencyContainer<PointInT, DataT>* pcl::octree::OctreePointCloudAdjacencyContainer<PointInT, DataT>::deepCopy() const [with PointInT = pcl::PointXYZRGBA; DataT = pcl::SupervoxelClustering<pcl::PointXYZRGBA>::VoxelData]’:
../segmentation/include/pcl/segmentation/supervoxel_clustering.h:134:13: warning: ‘<anonymous>.pcl::SupervoxelClustering<pcl::PointXYZRGBA>::VoxelData::distance_’ is used uninitialized in this function [-Wuninitialized]
       class VoxelData
             ^~~~~~~~~
../segmentation/include/pcl/segmentation/supervoxel_clustering.h:134:13: warning: ‘<anonymous>.pcl::SupervoxelClustering<pcl::PointXYZRGBA>::VoxelData::idx_’ is used uninitialized in this function [-Wuninitialized]
../segmentation/include/pcl/segmentation/supervoxel_clustering.h: In member function ‘pcl::octree::OctreePointCloudAdjacencyContainer<PointInT, DataT>* pcl::octree::OctreePointCloudAdjacencyContainer<PointInT, DataT>::deepCopy() const [with PointInT = pcl::PointXYZRGB; DataT = pcl::SupervoxelClustering<pcl::PointXYZRGB>::VoxelData]’:
../segmentation/include/pcl/segmentation/supervoxel_clustering.h:134:13: warning: ‘<anonymous>.pcl::SupervoxelClustering<pcl::PointXYZRGB>::VoxelData::distance_’ is used uninitialized in this function [-Wuninitialized]
       class VoxelData
             ^~~~~~~~~
```

--- a/segmentation/include/pcl/segmentation/supervoxel_clustering.h
+++ b/segmentation/include/pcl/segmentation/supervoxel_clustering.h
@@ -139,6 +139,8 @@ namespace pcl
             rgb_ (0.0f, 0.0f, 0.0f),
             normal_ (0.0f, 0.0f, 0.0f, 0.0f),
             curvature_ (0.0f),
+            idx_(0),
+            distance_(0),
             owner_ (nullptr)
             {}
 

